### PR TITLE
added task and manuscript fields to task

### PR DESCRIPTION
**Pull request by kobiCohen**

### CODE REVIEW

File: client/app/components/pages/manuscripts.component.ts [hunk 1]
```diff
@@ -121,6 +123,11 @@ export class ManuscriptsComponent {
 					this.tService.addTask(t).subscribe(
 						r=>{							
 							alert("task created succesfuly")
+							this.tService.getTasks({}).subscribe(
```
Comment (sahare92) [inline client/app/components/pages/manuscripts.component.ts]: what is the purpose of this?

File: client/app/models/Task.ts [hunk 2]
```diff
@@ -2,8 +2,10 @@ import { User } from './User';
 import { PageAnnotation } from './PageAnnotation';
 
 export class Task {
-	_id: Number;
+	_id: string;
```
Comment (sahare92) [inline client/app/models/Task.ts:5]: nice that you noticed!

File: models/task.js [hunk 3]
```diff
@@ -37,6 +37,20 @@ var taskSchema = mongoose.Schema({
...
+		ref: 'Manuscript',
+		required: true 
+	}
+	,
+	page: 
+	{
+	type: mongoose.Schema.Types.ObjectId,
+	ref: 'Page',
+	required: true 
+	}
```
Comment (sahare92) [inline models/task.js]: fix the alignment to look like the rest

**Review by sahare92 — CHANGES_REQUESTED** [review]
good job! small changed needed

File: client/app/components/pages/manuscripts.component.ts (hunk 1 above)
Comment (kobiCohen) [reply client/app/components/pages/manuscripts.component.ts]: log to see that it was created successfuly i will delete

**Pull request merged**